packages/git/src/browser/history/git-history-widget.ts

```diff
@@ -240,6 +255,25 @@ export class GitHistoryWidget extends GitBaseWidget<GitHistoryListNode> implemen
         }, content);
     }
 
+    protected async openDetailWidget(commit: GitCommitNode) {
+        const range = {
+            fromRevision: commit.commitSha + "~1",
+            toRevision: commit.commitSha
+        };
+        const widget = await this.widgetManager.getOrCreateWidget<GitCommitDetailWidget>(GIT_COMMIT_DETAIL,
```

Comment:
@svenefftinge Should not we introduce the widget open handler for it that the commit details can be opened from other places as well?

Reply:
Yes, makes sense.